interpreter/Makefile

```diff
@@ -9,131 +9,39 @@
 
 # Configuration
 
-NAME =		wasm
-UNOPT = 	$(NAME).debug
-OPT =   	$(NAME)
-LIB =		$(NAME)
-ZIP =		$(NAME).zip
-JSLIB =		wast
-WINMAKE =	winmake.bat
-
-DIRS =		util syntax binary text valid runtime exec script host main tests
-LIBS =		
-FLAGS = 	-lexflags -ml -cflags '-w +a-4-27-42-44-45-70 -warn-error +a-3'
-OCBA =		ocamlbuild $(FLAGS) $(DIRS:%=-I %)
-OCB =		$(OCBA) $(LIBS:%=-libs %)
-JSO =		js_of_ocaml -q --opt 3
-JS =		# set to JS shell command to run JS tests, empty to skip
+OPT =		wasm.exe
+ZIP =		wasm.zip
+JSLIB =		wast.bc.js
 
+BUILDDIR = _build/default
 
-# Main targets
+JS =		# set to JS shell command to run JS tests, empty to skip
 
-.PHONY:		default opt unopt libopt libunopt jslib all land zip smallint dunebuild
 
-default:	opt
-debug:		unopt
-opt:		$(OPT)
-unopt:		$(UNOPT)
-libopt:		_build/$(LIB).cmx _build/$(LIB).cmxa
-libunopt:	_build/$(LIB).cmo _build/$(LIB).cma
-jslib:		$(JSLIB).js
-all:		unopt opt libunopt libopt test
-land:		$(WINMAKE) all
-zip: 		$(ZIP)
-smallint:	smallint.native
-ci:			land jslib dunebuild
+# Main targets
 
-dunebuild:
-	dune build
+.PHONY:		default opt jslib all zip smallint
 
+default:	$(OPT)
+jslib:		$(JSLIB)
+all:		$(OPT) test
+zip:		$(ZIP)
+smallint:	smallint.exe
+ci:		all jslib
 
 # Building executable
+.PHONY:		wasm.exe
+wasm.exe:
+	dune build $@
 
-empty =
-space =		$(empty) $(empty)
-comma =		,
-
-.INTERMEDIATE:	_tags
-_tags:
-		echo >$@ "true: bin_annot"
-		echo >>$@ "true: debug"
-		echo >>$@ "<{$(subst $(space),$(comma),$(DIRS))}/*.cmx>: for-pack($(PACK))"
-
-$(UNOPT):	main.byte
-		mv $< $@
-
-$(OPT):		main.native
-		mv $< $@
-
-.PHONY:		main.byte main.native
-main.byte:	_tags
-		$(OCB) -quiet $@
-
-main.native:	_tags
-		$(OCB) -quiet $@
-
-.PHONY:		smallint.byte smallint.native
-smallint.byte: _tags
-		$(OCB) -quiet $@
-smallint.native: _tags
-		$(OCB) -quiet $@
-
-
-# Building library
```

Comment:
This is used to create the Wasm opam package. Perhaps it can be simplified, but we still need it.

Reply:
If you're OK with it, I can bring back proper opam support in another PR, but it requires more fixes (e.g. having a `public_name` and not simply a `name` etc.).